zerver/migrations/0001_initial.py

```diff
@@ -36,10 +37,47 @@ class Migration(migrations.Migration):
 
     initial = True
 
+    zulip_postgres_dictionaries_unavailable = os.getenv("POSTGRES_MISSING_DICTIONARIES", False)
```

Comment:
Should we stash this in `settings` instead of an environment variable, in case we need to modify the text search configuration in a future migration?

Reply:
Yeah, I suppose we could use the `sed` approach to put it into settings like we do `ZULIP_ADMINISTRATOR`, but a setting read from `zulip.org` feels more natural for me?